certification/artifacts/artifacts_test.go

```diff
@@ -1,41 +1,30 @@
-package artifacts
+package artifacts_test
```

Comment:
It can be, but I made it `artifacts_test` because the tests I executed don't require internal knowledge of the workings of the package. This is the Go implementation of "black box testing".